lib/wasi/src/state/types.rs

```diff
@@ -375,80 +370,6 @@ pub(crate) fn poll(
 
 pub trait WasiPath {}
 
-/// For piping stdio. Stores all output / input in a byte-vector.
```

Comment:
We could consider doing a re-export of the new pipe with the old pipe so older code of those that use Wasmer will still compile - or at least a deprecation period.

Reply:
Yeah so just `use WasiPipe as Pipe;`

Reply:
i think its public right? so might need to be a public export.
also... not sure if you can put a deprecated attribute on an export - if you can then you can tell them to use the other one instead